libkbfs/folder_branch_ops.go

```diff
@@ -2241,6 +2247,31 @@ func (fbo *folderBranchOps) doMDWriteWithRetry(ctx context.Context,
 				if err = fbo.cr.Wait(ctx); err != nil {
 					return err
 				}
+			} else if isRevisionConflict(err) {
```

Comment:
Why is this code here, instead of right after the `PutUnmerged` call in `finalizeMDWriteLocked`? That seems like a more logical place to put it. And then you can avoid making revision conflict a retriable error.